goc/config-osg/etc/cvmfs/config.d/cms.osgstorage.org.conf

```diff
@@ -1,3 +1,3 @@
-CVMFS_EXTERNAL_URL="https://srm.unl.edu:1094/"
-CVMFS_SERVER_URL="https://hcc-cvmfs-repo.unl.edu/cvmfs/@fqrn@"
```

Comment:
Please keep `CVMFS_SERVER_URL` and `CVMFS_HTTP_PROXY` as they were -- point directly at `hcc-cvmfs-repo`.

I would like to eventually re-add namespace protection to this repo; I'm a bit afraid that I'll add it just in time for this change to go out.  Regardless, for the first month or so, I'm not overly concerned about scale - we'll mostly be working to create understanding within CMS of the repo.

The new setting for `CVMFS_EXTERNAL_URL` is correct.

Reply:
Why not allow use of the site proxies for the metadata?  I guess one reason might be that some proxies restrict the destination to known servers.

Reply:
I can't get it to work with the https CVMFS_SERVER_URL.  It works with http, is it OK to use that?

Reply:
Port 443 on hcc-cvmfs-repo.unl.edu gives Connection refused

Reply:
Ah sorry - yes, I suppose we have to do a change regardless since we don't export 443 currently.

Let's keep the PR as-is and move forward.

Reply:
In offline discussion Brian & I agreed to use http on hcc-cvmfs-repo in CVMFS_SERVER_URL for now.